msteams-platform/m365-apps/actions-in-m365.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Actions in Microsoft 365
+description: In this article, learn more about the function of Actions and its use cases. 
+ms.date: 11/16/2023
+ms.author: mosdevdocs
+author: v-bvishnu
+ms.topic: Conceptual
+ms.subservice: m365apps
+---
+# Actions in Microsoft 365
+
+> [!NOTE]
+>
+> * Actions for Microsoft 365 is available in [public developer preview](../resources/dev-preview/developer-preview-intro.md).
+>
+> * Actions is supported for Microsoft 365 apps on web and desktop clients only and isn't supported on Outlook and Microsoft Teams.
+
+Actions enable your app to integrate seamlessly into the user's workflow and guide users to your app based on their intent and the content at hand, making task completion more efficient.
+Actions help you enhance your app's visibility and engagement with minimal development effort and reduce the need to switch contexts across various Microsoft 365 apps.
```

Comment:
This is talking as you is the developer. However the second half "reduce the need to switch contexts across various Microsoft 365 apps" is actions value to end users. It will be better to say "help your users to reduce context switch ..."

Reply:
Updated as "Actions help you enhance your app's visibility and engagement with minimal development effort and reduce the need for users to switch contexts across various Microsoft 365 apps."